database/migration/migrateData.ts

```diff
@@ -1131,20 +1131,30 @@ STABLE;
     ALTER TABLE user_organisation ADD CONSTRAINT user_organisation_user_id_organisation_id_key UNIQUE (user_id, organisation_id);
     `)
 
```

Comment:
All below needs to go in statement for versionLessThan 0.4.6

I suggest doing this after rebasing this `feature` branch from `develop` and merging here (there are quite a lot of migration code for 0.4.6)